lib/chef/telemetry/decision/prompt.rb

```diff
@@ -0,0 +1,101 @@
+# Use same support libraries as license-acceptance
+require "tty-prompt"
+require "pastel"
+require "timeout"
+require "chef-config/windows"
+
+class Chef
+  class Telemetry
+    class Decision
+      # Represents a decision made by interactively prompting the user
+      class Prompt
+
+        attr_reader :logger, :output, :input
+
+        PASTEL = Pastel.new
+        BORDER = "+---------------------------------------------+".freeze
+        CHECK = PASTEL.green(ChefConfig.windows? ? "√" : "✔")
+        X_MARK = PASTEL.red(ChefConfig.windows? ? "x" : "×")
+        CIRCLE = PASTEL.green(ChefConfig.windows? ? "O" : "◯")
+
+        def initialize(cfg)
+          @logger = cfg[:logger]
+          @output = cfg[:output]
+          @input = cfg[:input] || STDIN
+        end
+
+        def prompt(dir, persistor)
+          logger.debug "Prompting for opt-in/out..."
+
+          output.puts <<~EOM
+            #{BORDER}
+            Share Data with Chef
```

Comment:
Maybe a `:` at the end of this line since it is a header for the following sentence?